### PR TITLE
Core notes Aug 3rd 2015

### PR DESCRIPTION
**Pull request by mixonic**

### CODE REVIEW

**Comment by rwjblue** [comment]
Thank you!

File: ember.js/2015-08/august-3.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+* ember-registry-container-reform
+
+##### Important Issues:
+
+* https://github.com/emberjs/ember.js/pull/11315 - Changing layout in the
+* https://github.com/emberjs/ember.js/pull/11225 - Two way bindings with setter CP's are not working.
+
+#### Minutes
+
+Attendees: Matt, Robert, Tom, Yehuda, Stef, Kris, ?
```
Comment (stefanpenner) [inline ember.js/2015-08/august-3.md]: i was unable to attend, although that likely doesn't matter much :P

Comment (mixonic) [inline ember.js/2015-08/august-3.md]: I could hear your voice in my head though....

Comment (mixonic) [inline ember.js/2015-08/august-3.md]: fixed in a push.

**Pull request merged**